### PR TITLE
20230328 01

### PR DESCRIPTION
**Pull request by hkortti**
New instructions and index entries for abandoned servers, exposed services and expired certificates.

### CODE REVIEW

File: docs/types/expired-certificate.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+into use. According to those logs, a certificate assigned to a host in your organization has expired, and it has not yet been 
+renewed. When a certificate expires, services relying on TLS on that host no longer work correctly, and those services may 
+not be reachable by their intended users.
+
+## Verifying the issue
+
+You can use a public Certificate Transparency logs search engine such as crt.sh to check the status of your certificate. 
+Replace the hostname in the query string with your hostname. 
+* https://crt.sh/?q=host.example.com
+
```
Comment (evilon) [inline docs/types/expired-certificate.md:17]: Should we mentio that  it is possible that old expiring certificate is being replaced with wildcard cert. The trouble is, that we can't probably go as far as instructing how the actual situation is verified with different browsers.

Comment (hkortti) [reply docs/types/expired-certificate.md:17]: Added a mention

**Review by evilon — APPROVED** [review]

**Review by evilon — APPROVED** [review]
LGTM

**Comment by evilon** [comment]
LGTM

**Pull request merged**